src/renderer/missionWindow/MissionWindow.tsx

```diff
@@ -201,12 +318,54 @@ export default class MissionWindow extends Component<ThemeProps, State> {
     this.setState({ activeVehicleMapping: newActiveVehicleMapping });
   }
 
-  private updateInformation(information: MissionInformation.Information): void {
-    const { information: currentInformation } = this.state;
-    const newInformation = currentInformation;
+  private postStartMissions(): void {
```

Comment:
same here, why are all these functions moved around?

Reply:
yeah you actually dont need to move any of these functions around, i checked it out for myself just now.